### DISCUSSION TITLE
Breaking change with watch since 7.5.2?

### DISCUSSION THREAD

**Discussion by medihack** [react-hook-form/react-hook-form#5482] [Q&A]
In `react-hook-form` v7.5.2 I used `watch` this way
```
watch((data, { name, type, value }) => {
  ..
}
```
and `value` contained the correct changed value. With v7.7.1 `value` is `undefined` and not even mentioned [in the docs anymore](https://react-hook-form.com/api/useform/watch). But TS still thinks it is ok to use `value` in the callback (no typescript error). I wonder a bit, is this documented somewhere? I searched the release notes, but couldn't find anything about this change.

**Comment by bluebill1049** [comment]
raise an issue and provide reproducible codesandbox.
